Sources/PortalSwift/Portal.swift

```diff
@@ -349,6 +355,30 @@ public class Portal {
     )
   }
 
+  /// You should only call this function if you are upgrading from v3 to v4.
```

Comment:
is it possible to make this comment a decorator that pops up like a deprecated symbol or not?

Reply:
![Screenshot 2024-07-31 at 4 34 37 AM](https://github.com/user-attachments/assets/56d51012-a7eb-4176-9dc4-a897650a9513)

Adding a comment with `///` 3 forward slashes is adding documentation to the developer appears with the Xcode auto-complete by default.